int64 speed_limit